configs/_base_/models/twins_fpn.py

```diff
@@ -0,0 +1,36 @@
+# model settings
```

Comment:
rename ``configs/_base_/models/twins_fpn.py to `configs/_base_/models/twins_pcpvt-small_fpn.py`